app/models/patient.rb

```diff
@@ -252,6 +252,14 @@ def has_special_circumstances
     !!has_circumstance
   end
 
+  def archive_date
+    if fulfillment.audited?
+      initial_call_date + 3.months
```

Comment:
also, could we add a comment here from your email explaining the logic here? copying and pasting straight from the email is fine:

```
 - If a patient fulfillment is ticked off as audited, they will archive 3 months after the patient's initial call date
    - if we're already past 3 months later when the audit happens, it will archive that night
 - if a patient is waiting for audit they will not archive until a year after their initial call date, but then they will archive regardless of audit status.
```